### PR TITLE
Fixed issue IoT Core wrong topic on publish #31

### PR DESCRIPTION
**Pull request by hugAdmin**
Topic can now be passed in as an argument

### CODE REVIEW

**Comment by hugAdmin** [comment]
I signed it!

File: iot/mqtt-ciotc/mqtt_ciotc.c [hunk 1]
```diff
@@ -29,33 +29,33 @@
...
   char* rootpath;
-  enum { topic_maxlen = 256, topic_size };
   char topic[topic_size];
   char* payload;
-  char* algorithm;
 } opts = {
   .address = "ssl://mqtt.googleapis.com:8883",
   .clientid = "projects/{your-project-id}/locations/{your-region-id}/registries/{your-registry-id}/devices/{your-device-id}",
   .deviceid = "{your-device-id}",
-  .keypath = "ec_private.pem",
```
Comment (gguuss) [inline iot/mqtt-ciotc/mqtt_ciotc.c]: Any reason we're removing support for RSA keypairs?

File: iot/mqtt-ciotc/mqtt_ciotc.c [hunk 2]
```diff
@@ -85,22 +85,12 @@ static void GetIatExp(char* iat, char* exp, int time_size) {
   }
 }
 
-static int GetAlgorithmFromString(const char *algorithm) {
```
Comment (gguuss) [inline iot/mqtt-ciotc/mqtt_ciotc.c]: Note we support RSA as well as ES256 (EC). We may want to leave this in.

**Comment by gguuss** [comment]
@hugAdmin I'm still not seeing your GitHub username in our list of CLA signers, can you make sure to sign up with the username on the PR?

**Comment by tmatsuo** [comment]
Closing due to inactivity

**Pull request closed without merging**